thorlcr/thorutil/thormisc.cpp

```diff
@@ -1732,7 +1732,12 @@ bool hasTLK(IDistributedFile &file, CActivityBase *activity)
         part.getFilename(rfn);
         StringBuffer filename;
         rfn.getPath(filename);
-        Owned<IKeyIndex> index = createKeyIndex(filename, 0, false, 0);
+        StringBuffer planeName;
+        constexpr size32_t bufferSize1mb = 0x100000;
+        size32_t bufferSize = bufferSize1mb;
+        if (findPlaneFromPath(filename, planeName))
+            bufferSize = (size32_t)getPlaneAttributeValue(planeName, BlockedSequentialIO, bufferSize);
```

Comment:
it's obscure edge case that will enter this code, correct to do so, but of limited benefit.
NB: as this code stands, unlike the other changes, it checks the plane setting only, not the expert component setting. I think all places should only do that too.

Reply:
should use 'findPlaneAttrFromPath'